mesonbuild/coredata.py

```diff
@@ -211,6 +211,7 @@ def get_builtin_option_default(optname):
         'datadir'           : [ UserStringOption, 'Data file directory.', 'share' ],
         'mandir'            : [ UserStringOption, 'Manual page directory.', 'share/man' ],
         'localedir'         : [ UserStringOption, 'Locale data directory.', 'share/locale' ],
+        'sysconfdir'        : [ UserStringOption, 'Sysconf data directory.', '/etc' ],
```

Comment:
If the default prefix is `/usr/local`, the default sysconfdir should be `/usr/local/etc`.